src/components/Theme/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { kebabCase } from '~utils/string'
+import { UICustomisationOptions } from '~types/ui-customisation-options'
+
+export const setUICustomisations = (customUI: UICustomisationOptions): void => {
+  const sdkCustomisations = Object.keys(customUI).map(
+    (property: keyof typeof customUI) =>
+      `--osdk-${kebabCase(property)}: ${customUI[property]};`
+  )
+  if (sdkCustomisations.length > 0) {
+    const style = `
+      <style>
+          :root {
+            ${sdkCustomisations.join('\n')}
+          }
+      </style>`
+    document.head.insertAdjacentHTML('beforeend', style)
+  }
+}
+
+export const isButtonGroupStacked = (): boolean =>
+  !!JSON.parse(
+    getComputedStyle(document.body).getPropertyValue(
+      '--osdk-button-group-stacked'
+    )
+  )
```

Comment:
💅🏻  [nitpick]

That CSS variable should be a string right? Either `false` or `true`.
Instead of the expensive `JSON.parse` you can just check the value.

```suggestion
export const isButtonGroupStacked = (): boolean =>
  'true' ===
  getComputedStyle(document.body)
    .getPropertyValue('--osdk-button-group-stacked')
    .trim()
```

Reply:
Yes, the CSS variable should be a string. Will double check this locally just to be sure.